tests/test_load_link16.py

```diff
@@ -0,0 +1,162 @@
+        filename = "GEV_09-05-2021T16-10-00.raw-PPLI_201.csv"
+        assert Link16Importer.extract_timestamp(filename) == "09-05-2021T16-10-00"
+
+    def test_extract_timestamp_relative_v2(self):
+        filename = "GEV_12-09-2021T09-25-00.raw-SLOTS_JMSG.csv"
+        assert Link16Importer.extract_timestamp(filename) == "12-09-2021T09-25-00"
+
+    def test_convert_timestamp_ambiguous_day_month(self):
+        timestamp = "09-05-2021T16-10-00"
+        assert Link16Importer.timestamp_to_datetime(timestamp) == datetime(2021, 5, 9, 16, 10, 0)
+
+    def test_convert_timestamp_missing(self):
+        timestamp = ""
+        assert Link16Importer.timestamp_to_datetime(timestamp) is False
+
+    def test_convert_timestamp_invalid(self):
+        timestamp = "123456-7-8-9"
+        assert Link16Importer.timestamp_to_datetime(timestamp) is False
+
+    def test_process_link16_v1_data(self):
+        processor = FileProcessor(archive=False)
+        processor.register_importer(Link16Importer())
+
+        # check states empty
+        with self.store.session_scope():
+            # there must be no states at the beginning
+            states = self.store.session.query(self.store.db_classes.State).all()
+            self.assertEqual(len(states), 0)
+
+            # there must be no platforms at the beginning
+            platforms = self.store.session.query(self.store.db_classes.Platform).all()
+            self.assertEqual(len(platforms), 0)
+
+            # there must be no datafiles at the beginning
+            datafiles = self.store.session.query(self.store.db_classes.Datafile).all()
+            self.assertEqual(len(datafiles), 0)
+
+        # parse the data
+        processor.process(DATA_PATH_V1, self.store, False)
+
+        # check data got created
+        with self.store.session_scope():
+            # there must be states after the import
+            states = self.store.session.query(self.store.db_classes.State).all()
+            self.assertEqual(len(states), 8)
+
+            # there must be platforms after the import
+            platforms = self.store.session.query(self.store.db_classes.Platform).all()
+            self.assertEqual(len(platforms), 7)
+
+            # there must be one datafile afterwards
+            datafiles = self.store.session.query(self.store.db_classes.Datafile).all()
+            self.assertEqual(len(datafiles), 1)
+
+            results = (
+                self.store.session.query(self.store.db_classes.State)
+                # Elevation == 4222 ft
+                .filter(func.round(self.store.db_classes.State.elevation, 1) == 1286.9)
+                .order_by(self.store.db_classes.State.time)
+                .all()
+            )
+            assert len(results) == 3
+            assert results[0].time == datetime(2021, 5, 9, 1, 46, 38, 100000)
+            assert results[1].time == datetime(2021, 5, 9, 2, 40, 47, 400000)
+            assert results[2].time == datetime(2021, 5, 9, 3, 16, 35, 800000)
```

Comment:
@mew-nsc in the above lines, it would be useful to also test that we have parsed location, course & speed too.  

Aaah, and maybe check the name for one of the new platform entries too - I guess it should be the `STN` value.